Troubleshooting/README.md

```diff
@@ -1,15 +1,26 @@
 Troubleshooting Guide
 ---------------------
-This guide lists issues you may run into on your Gluster setup and provides information on how to recover from them.
-You can also reach out to the Gluster community for help.
+This guide describes some commonly seen issues and steps to recover from them.
+If that doesn’t help, you can also reach out to the [Gluster community](https://www.gluster.org/community/), in which case the guide also describes what information needs to be provided in order to debug the issue. At minimum, we need the version of gluster running and, in most cases, the output of `gluster volume info`.
 
 
-The first step in troubleshooting is identifying the problem and which process or component to look at.
-The following tools/utilities are helpful.
+### Where Do I Start?
+
+See if the issue is already listed in the component troubleshooting sections
+
+- [CLI and Glusterd Issues](./troubleshooting-glusterd.md)
+- [Resolving Split brains](./resolving-splitbrain.md)
+- [Geo-replication Issues](./troubleshooting-georep.md)
+
+
+If that didn't help, here is how to debug further.
+
+When things go wrong, identifying the problem and getting the necessary information in order to diagnose it is the first step in troubleshooting your Gluster setup. As Gluster operations involve interactions between multiple processes, this can involve multiple steps.
+
+### What Happened?
```

Comment:
Suggestion: This section doesn't seem to contain much yet. You may find it hard to reason about generic failure such as "an operation failed" and map it to a specific troubleshooting guide.  The `high memory usage` link can be part of the list described above.

Reply:
I plan to include information as to how to determine the logs to look at as well as what information to provide.